src/server/browserServer.ts

```diff
@@ -16,13 +16,11 @@
 
```

Comment:
Looks empty now

Reply:
Yeah, I'll follow up with merging it into Browser probably.